Misc/NEWS.d/next/Core and Builtins/2024-03-21-09-57-57.gh-issue-117114.Qu-p55.rst

```diff
@@ -0,0 +1 @@
+Added POSIX implementation for :func:`os.path.isdevdrive`.
```

Comment:
```suggestion
Make :func:`os.path.isdevdrive` available on all platforms. For those that do not offer Dev Drives, it will always return ``False``.
```